include/z64interface.h

```diff
@@ -151,27 +151,27 @@ typedef struct InterfaceContext {
 #define A_BUTTON_X 186
 #define A_BUTTON_Y 9
 
-#if PLATFORM_N64
-#define A_BUTTON_R 90
-#define A_BUTTON_G 90
-#define A_BUTTON_B 255
-#else
+#if PLATFORM_GC
 #define A_BUTTON_R 0
 #define A_BUTTON_G 200
 #define A_BUTTON_B 50
+#else
+#define A_BUTTON_R 90
+#define A_BUTTON_G 90
+#define A_BUTTON_B 255
 #endif
```

Comment:
personally I'd prefer to see the two color sets in chronological order, so `!PLATFORM_GC` first